docs/Frameworks.md

```diff
@@ -51,7 +51,7 @@ it('should test something', () => {
 })
 ```
 
-If you want to run something asynchronously, you can either use the [`browser.call`](call) command or [custom commands](CustomCommands.md).
+If you want to run something asynchronously, you can either use the [`browser.call`](api/browser/call.html) command or [custom commands](CustomCommands.md).
```

Comment:
This will be transformed by Docusarus, no need for this change

Reply:
I didn't change them.. This is exactly how it is on the repo. I am looking into why this looks like it changed. :S

Reply:
nevermind then